### ISSUE TITLE
[Question] Limited lesion proposals

### ISSUE THREAD

**Issue by code8865** [MIC-DKFZ/nnDetection#305]
### :question: Question

Hi,

I'm working on a relatively small dataset for renal tumor detection (50 patients, 1 tumor each). Using nnDetection, I currently reach ~80% sensitivity in CV. Achieving a higher sensitivity even with low diagnostic thresholds seems out of reach, as the model fails to propose any bounding box for some patients.

Are there recommended settings or strategies in nnDetection to encourage more lesion proposals—ideally boosting sensitivity at the cost of precision?

Thanks in advance!

**Comment by andres2631996** [comment]
Are you already applying model ensembling on a separate test set? If not, ensembling the cross-validation models after running consolidation could maybe provide more boxes, since there are more models involved. 

You could also increase the parameter "oversample_foreground_percent" in "nndet/conf/train/v001.yaml" to 0.75, so that the network gets even more used to dealing  with tumors and is more prone to output more predictions, but at the cost of increased overfitting. 

You could also increase the ratio of positive anchors in "positive_fraction" in "nndet/conf/train/v001.yaml" to 0.5, to influence on the Hard Negative Mining process.
